app/assets/stylesheets/sections/dashboard.scss

```diff
@@ -120,6 +120,15 @@
   }
 }
 
+.dash-new-group {
+  background: $bg_success;
```

Comment:
Name of variable `bg_success` should be written in all lowercase letters with hyphens instead of underscores